### DISCUSSION TITLE
Issues since update 1.25.2

### DISCUSSION THREAD

**Discussion by RulHal88** [teslamate-org/teslamate#2362] [Q&A]
Since the update to version 1.25.2, the page http://your-ip-address:4000 is no longer accessible.
Error: ip-address refused a connection.
My Raspberry Pi 3 has been running very slowly since the update, services no longer work (e.g. CUPS), Putty access no longer works.
After a reboot it runs again for a short time, then the game starts over.

**Comment by cwanja** [comment]
I would suggest checking out #2302 - specifically around either updating to `Bullseye` or backporting the service file.

**Reply by cwanja** [reply]
Specifics around backporting https://github.com/adriankumpf/teslamate/issues/2302#issuecomment-1011362938

**Comment by RulHal88** [comment]
Thanks for the help.
I'm downgrading to Teslamate 1.25.0

**Reply by cwanja** [reply]
That will fix it immediately. But to upgrade to any future version, you will need to do the upgrade or backporting mentioned above. See the release notes of 1.26.
